scp/lib/ui/views/mentor_search/mentor_page.dart

```diff
@@ -343,17 +344,12 @@ class MentorDetails extends StatelessWidget {
         });
   }
 
-  void saveContact(Contact newContact) async {
-    NativeContactDialog.addContact(newContact).then((result) {
-      // NOTE: The user could cancel the dialog, but not add
-      // them to their addressbook. Whether or not the user decides
-      // to add [contactToAdd] to their addressbook, you will end up
-      // here.
-
-      print('add contact dialog closed.');
+  void saveContact(Contact contact) async {
+    // save contact
+    final result = FlutterContacts.insertContact(contact).then((value) {
+      print('add contact dialog closed');
     }).catchError((error) {
-      // FlutterError, most likely unsupported operating system.
-      print('Error adding contact!');
+      print('pata nai bhai kya error hai');
```

Comment:
If the plugin is creating the issue, then try using [contacts_service](https://pub.dev/packages/contacts_service) plugin which has more likes and popularity.

Reply:
Not an updated plugin. Has the old android embedding.